presto-product-tests/bin/run_on_docker.sh

```diff
@@ -174,8 +174,15 @@ elif [[ "$ENVIRONMENT" == "multinode-tls" ]]; then
 fi
 
 CLI_ARGUMENTS="--server presto-master:8080"
-if [[ "$ENVIRONMENT" == "multinode-tls" ]]; then
-    CLI_ARGUMENTS="--server https://presto-master.docker.cluster:7778 --keystore-path /docker/volumes/conf/presto/etc/docker.cluster.jks --keystore-password 123456"
+CLI_KEYSTORE_ARGS="--keystore-path /docker/volumes/conf/presto/etc/docker.cluster.jks --keystore-password 123456"
+if [[ "$ENVIRONMENT" == "singlenode-ldap" ]]; then
+    CLI_ARGUMENTS="--server https://presto-master.docker.cluster:8443 ${CLI_KEYSTORE_ARGS}"
+fi
+if [[ "$ENVIRONMENT" == "multinode-tls" || "$ENVIRONMENT" == *kerberos* ]]; then
+    CLI_ARGUMENTS="--server https://presto-master.docker.cluster:7778 ${CLI_KEYSTORE_ARGS}"
+fi
+if [[ "$ENVIRONMENT" == *kerberos* ]]; then
+    CLI_ARGUMENTS="${CLI_ARGUMENTS} --enable-authentication --krb5-config-path /etc/krb5.conf --krb5-principal presto-client/presto-master.docker.cluster@LABS.TERADATA.COM --krb5-keytab-path /etc/presto/conf/presto-client.keytab --krb5-remote-service-name presto-server --krb5-disable-remote-service-hostname-canonicalization"
```

Comment:
is `--enable-authentication` specific to kerberos?

would be best not to have `--krb5-disable-remote-service-hostname-canonicalization` in tests, but this might be out of scope for this PR

Reply:
Excerpt from docs.
--enable-authentication | Enables Kerberos authentication.
-- | --

Regarding `--krb5-disable-remote-service-hostname-canonicalization`, I would better not to touch this one now. I had to enable something like this to make simba jdbc driver work with kerberos.